Front/pharmacy/src/views/header/PatientHeader.vue

```diff
@@ -9,7 +9,23 @@
         <li class="nav-item"><a href="#" class="nav-link">Žalbe</a></li>
         <b-nav-item-dropdown text="Mirko Mirković" right>
           <b-dropdown-item href="#"><router-link to="/patient-profile">Moj profil</router-link></b-dropdown-item>
-          <b-dropdown-item href="#">Odjava</b-dropdown-item>
+          <b-dropdown-item href="#" @click="logout" >Odjava</b-dropdown-item>
         </b-nav-item-dropdown>
     </ul>
 </template>
+
+<script>
+export default {
+  methods:{
+    logout : function(){
+      localStorage.setItem("UserRole","ROLE_UNREGISTERED")
+      localStorage.removeItem("UserId")
+      localStorage.removeItem("JWT")
+      this.$router.push("Registration");
+      window.location.reload()
+      
+    }
```

Comment:
logout ne moze ostati samo na ovome. Mora se i u backu resiti.

Reply:
okej, to je takodje privremeno 
dok se ne odradi u back delu